### ISSUE TITLE
Binding rule should supports importing module without importing types

### ISSUE THREAD

**Issue by atsu85** [MeirionHughes/aurelia-template-lint#83]
When I have class that imports module without importing types:
```TypeScript
    import "some-module";
    export class Foo{
      existing:string;
    }
```
and view:
```html
    <template>
      ${existing}
      ${missing}
    </template>
```
and i have enabled using binding rule and reporting errors:
```JavaScript
config.useRuleAureliaBindingAccess = true;
config.debug = true;
```
then i see an unexpected error:

> Cannot read property 'namedBindings' of undefined'.

instead of
> cannot find 'missing' in type 'Foo'

**Comment by MeirionHughes** [comment]
Cool thanks; Sooo many use-cases. :S

**Comment by atsu85** [comment]
PR is ready: https://github.com/MeirionHughes/aurelia-template-lint/pull/84

**Comment by MeirionHughes** [comment]
what does `import "some-module"` actually import? the default export of the module or everything from the module?

**Comment by atsu85** [comment]
Neither, as I understand. It just loads module (that can potentially pollute global namespace or execute some other code)

**Comment by MeirionHughes** [comment]
okaydoky... that one confused me; actually compiled it to see what it did. just requires the modules (presumably to run it). fair enough.
